### DISCUSSION TITLE
Reading from right to left

### DISCUSSION THREAD

**Discussion by Huang2326** [sumatrapdfreader/sumatrapdf#3961] [Q&A] [answered]
some Japanese comic book is arrange from right to left，but sumatraPDF default setting is reading from left to right.
how can I change the setting to "from right to left"?

**Comment by GitHubRulesOK** [comment, marked as answer]
look for the discussions about "Manga" mode but you can set your choice of key to toggle it and several times that has been shown for calling Cmd Toggle Manga Mode which is also available from CTRL + K 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/e57ebbf6-2070-4288-9e0e-e7046b2e1121)

for an example see https://github.com/sumatrapdfreader/sumatrapdf/issues/1264#issuecomment-1150241521
related to https://github.com/sumatrapdfreader/sumatrapdf/issues/1545

**Reply by Huang2326** [reply]
it's work.thanks
